ui/src/main/js/common/util/fieldModelUtilities.js

```diff
@@ -270,6 +270,7 @@ export const handleChange = (data, setData) => ({ target }) => {
     setData(newState);
 };
 
+// TODO: This name can be misleading, consider changing to handleDefaultChange to separate from old FieldModels
 export const handleTestChange = (testData, setTestData) => ({ target }) => {
```

Comment:
maybe we should call this method createDefaultInputChangeHandler because it's a function used to create an event handler.